core/templates/core/index.html

```diff
@@ -3,38 +3,30 @@
 {% block title %}Strona Główna{% endblock %}
 
 {% block content %}
-    <nav class="flex flex-wrap justify-between items-center mx-auto w-screen p-4 bg-gray-400 border-gray-800 bg-gray-900">
-            <a href="https://github.com/Gwardia-Czapli/GwardiaHub" class="hover:text-blue-100 text-gray-400 transition-colors duration-300" target="_blank">
+<div class="h-screen w-screen">
+    <nav class="flex flex-wrap justify-between items-center mx-auto w-screen p-2 border-gray-800 bg-gray-900">
+            <a href="https://github.com/Gwardia-Czapli/GwardiaHub" class="flex items-center hover:text-blue-100 text-gray-400 transition-colors duration-300 ml-5" target="_blank">
```

Comment:
Move `hover:text-blue-100` to image in line 9, so it's highlighted only when you hover over image, not when hovering over text